generated/nidaqmx/_library_interpreter.py

```diff
@@ -2228,11 +2228,52 @@ def get_ai_chan_cal_exp_date(self, task, channel_name):
 
     def get_analog_power_up_states(
             self, device_name, channel_name, channel_type):
-        raise NotImplementedError
+        state = []
+
+        args = [device_name]
+        argtypes = [ctypes_byte_str]
+
+        for index in range(None):
```

Comment:
I am guessing you know about this since you mention that fixes will be later, but FYI this isn't right. In mainline, we create a `Device` object and iteration the `ao_physical_chans`

Reply:
Lots of issues like this, just FYI

Reply:
Ahh, I like it. That makes sense!